.github/workflows/development.yaml

```diff
@@ -1,9 +1,17 @@
 name: Development
 on:
-  pull_request:
   push:
+    branches:
+      - '**' # every branch
+      - '!gh-pages' # exclude gh-pages branch
+      - '!stage*' # exclude branches beginning with stage
```

Comment:
The science team doesn't have a consistent approach to branch naming, often naming dev branches based on their new function, without a prefix. Does this mean that any push/PR on a dev branch will trigger doc builds for the main branch? If so, I might restrict this to main

Reply:
Discussed this in-person. The summary is that yes this will run a build on each PR but that this is still a valid test since it ensures reproducible installs i.e. no obvious errors. This image would normally be used for intergration/unit tests.